orc8r/gateway/python/magma/common/service_registry.py

```diff
@@ -85,6 +85,34 @@ def reset():
         """
         ServiceRegistry.get_registry()["services"] = {}
 
+    @staticmethod
+    def get_bare_rpc_channel(rootca_path=None):
```

Comment:
If we're gonna have this only target bootstrapper, then should reflect that in the func name